packages/server-wallet/src/wallet/store.ts

```diff
@@ -299,6 +314,40 @@ export class Store {
 
     return result;
   }
+
+  async createChannel(
+    constants: ChannelConstants,
+    appData: Bytes,
+    outcome: Outcome,
+    fundingStrategy: FundingStrategy
+  ): Promise<{outgoing: SyncState; channelResult: ChannelResult}> {
+    return await this.knex.transaction(async tx => {
+      const {channelId} = await createChannel(constants, fundingStrategy, tx);
+      return await this.signState(
+        channelId,
+        {
+          ...constants,
+          turnNum: 0,
+          isFinal: false,
+          appData,
+          outcome,
+        },
+        tx
+      );
+    });
+  }
+
+  async updateFunding(
+    channelId: string,
+    fromAmount: Uint256,
+    assetHolderAddress: Address
+  ): Promise<void> {
+    await Funding.updateFunding(this.knex, channelId, fromAmount, assetHolderAddress);
+  }
+
+  async nextNonce(signingAddresses: Address[]): Promise<number> {
+    return await Nonce.next(this.knex, signingAddresses);
+  }
```

Comment:
Why are these additions in scope for this PR?

Reply:
The wallet was in control of database models, which relied on a database connection, which is now contained within the store, so now the database models are also inside the store.

Reply:
e.g., 
![image](https://user-images.githubusercontent.com/1933029/94015164-80fb1f00-fd7a-11ea-992f-47a37ea208b1.png)